srcCxx/toc_coredx_dds/ShapeType_configurator_toc_coredx_dds.h

```diff
@@ -19,6 +19,138 @@ namespace DDS {
 #define DDSSEC_PROP_PERM_GOV_DOC       "dds.sec.access.governance"
 #define DDSSEC_PROP_PERM_DOC           "dds.sec.access.permissions"
 
+#define CDXSEC_PROP_SHARED_SECRET_ALGO "com.toc.sec.shared_secret_algorithm"
+
+
+static const char *get_qos_policy_name(DDS_QosPolicyId_t policy_id)
+{
+  return DDS_qos_policy_str(policy_id); // not standard...
+}
+
+
+class MyListenerClass : public DDS::DomainParticipantListener
+  {
+  public:
+    MyListenerClass() {};
+    ~MyListenerClass() {};
```

Comment:
```suggestion
    MyListenerClass() {}
    ~MyListenerClass() {}
```

Or use `=default;` since compilers should be new enough

Reply:
When I first read this I didn't realize it was in the `toc_*` directory -- I guess I shouldn't have an opinion on it (but I do anyway).